readme.md

```diff
@@ -458,6 +458,14 @@ Make sure that `SONATYPE_PASSWORD` uses proper escaping if it contains special
 characters as documented on
 [Travis Environment Variables](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings).
 
+If your sonatype account is new (created after Feb 2021), then the default server
+location inherited from the the `sbt-sonatype` plugin will not work, and you should add
+```
+sonatypeCredentialHost := "s01.oss.sonatype.org"
+sonatypeRepository := "https://s01.oss.sonatype.org/service/local"
+```
+to your root project
```

Comment:
Should this be scoped to the root project or `ThisBuild`?

Reply:
I... honestly don't know what it 'should' be. Best practice with `ThisBuild` in sbt has always eluded me. I think it's probably sanest to include it with the other publishing settings, so I'll try to suggest that in the copy...